src/selectors/markdownSelectors.test.js

```diff
@@ -0,0 +1,13 @@
+import { markdown } from '../selectors/markdownSelectors';
+
+describe.skip('markdown selectors', () => {
```

Comment:
why is this skipped?

Reply:
"We chose to focus elsewhere." - Marty Nelson, 2019.